frontend/src/components/AddMember/AddMember.scss

```diff
@@ -1,3 +1,19 @@
 .select, select{
     width: 100%
-}
\ No newline at end of file
+}
+
+.add-member .subtitle.is-2 {
+    font-size: 1rem;
+}
+
+.add-member .subtitle:not(:last-child) {
+    margin-bottom: 0.5rem;
+}
+
+.add-member .is-multiple .select:after {
+    display: none;
+}
+
+.add-member .is-multiple .select {
+    height: auto;
+}
```

Comment:
That's much better 🤘 

One small improvement would be to take advantage of [sass's nesting](https://sass-lang.com/documentation/style-rules#nesting)

Reply:
TIL  - cheers!